src/app/(logged-in)/challenges/page.tsx

```diff
@@ -2,40 +2,17 @@
 import * as React from "react";
 import {NextUIProvider} from "@nextui-org/system";
 import ChallengesTable from "../../../components/ChallengesTable";
-import { Challenges } from "@/types/challenges.types";
-import { BASE_API_URI } from "@/utils/constants";
-import { fetchData } from "@/utils/fetchData";
+import { useGetChallenges } from "@/hooks/challenges.hooks";
 
 export default function Page() {
-  const [data, setData] = React.useState<Challenges[]>([]);
-  const [loading, setLoading] = React.useState(true);
-  const [error, setError] = React.useState(null);
+  const curr_year = new Date().getFullYear();
+  const {isPending, error, data} = useGetChallenges(curr_year);
```

Comment:
Did they say they only want the challenges for the current year? Don't they want to be able to see all challenges? Since the challenges persist year after year, I assume they would want to see all created challenges unless specifying otherwise.